force-app/main/default/classes/DMLOperations.cls

```diff
@@ -114,8 +122,11 @@ public with sharing class DMLOperations {
     public static void updateAccountFields(Id accountId, String newName, String newIndustry) {
```

Comment:
This test for this is failing. Take a look at the code below, I think you are missing one thing.